### ISSUE TITLE
Revising your writing

### ISSUE THREAD

**Issue by cerdamejiaj** [ZachAndrews98/about-me#1]
In your latest commit 3987513, you have a few things you can improve. 

**Paragraph one**

- In " New York in order to closer to family" This sentence can be improved. You can say, " New York to be closer to the family."

   "in order to" can be wordy.   "family" is missing a determiner before it, adding an article can fix it like "the"

- In the phrase "On the other hand", you should add a comma after hand.

- You have "kinda" which is slang. You can use "quite" or "kind of"

- In "After about a year in Albany" you should add a comma after Albany.

- In "for an energetic five year" it should be years. 

- In " a year later" add a comma after later.

**Paragraph two**

- In the sentence "Although I had a new playmate I couldn't really do much with him between the naps "  you can remove "really" I don't think you need it in the sentence and is used many times in your writing.

- In "However I of course would still play with my toys giving them voices " you are missing a comma or two with the interrupter of course. "I, of course," or you can change it to "However, I would still play."

- In "However as I said TV was my main" add a comma after "however"

**Comment by mendezjw** [comment]
In your latest commit, I feel that these are somethings to improve on:

**Paragraph Three**

- In the sentence "It is also important to note that around this time my parents had bought me a Nintendo DS so I also had that while there." I think you can reword this to improve the flow and structure.

**Paragraph Four**

- In the sentence "As a result of this I abandoned trying to make a mod myself and went back to simply playing the game and using the mods other people had made for me." add a comma after "As a result of this"
